qsiprep/cli/parser.py

```diff
@@ -823,6 +824,12 @@ def parse_args(args=None, namespace=None):
             for session in sessions:
                 processing_groups.append([subject_id, [session]])
         else:
+            # We can now use sessions that have anatomical data, but no DWI
```

Comment:
This will process "unbiased" or "first" options using all of the anatomical data, I think.

Reply:
This looks right